azure-pipeline.yaml

```
@@ -8,6 +8,10 @@ resources:
   - repository: DevCentral/ni-central
     type: git
     name: ni-central
+  - repository: Tools
+    type: git
+    name: DevCentral/Tools
+    ref: 1034f8129f933c91c24abbd737e429590e3f23ff
```

I'm pretty concerned that this will break at some point in the future. Please put a comment here, explaining that we're using an old version of setup-venv.yml.

```suggestion
    # Use an old version of setup-venv.yml
    ref: 1034f8129f933c91c24abbd737e429590e3f23ff
```